src/pybind/mgr/cephadm/module.py

```diff
@@ -1605,6 +1605,16 @@ def _find_inv_for_host(hostname, inventory_list):
 
         return self._create_osd(cmds)
 
+    def show_drivegroups(self):
```

Comment:
```suggestion
    def get_drivegroups(self) -> List[DriveGroupSpec]:
```
Similar to https://docs.ceph.com/docs/master/mgr/orchestrator_modules/#orchestrator.Orchestrator.get_inventory ?